test_cromwell.sh

```diff
@@ -11,49 +11,48 @@ trap "shutdown" EXIT
 
 set -e
 
-EXIT_CODE=31
+EXIT_CODE=1
 PROGNAME="$(basename $0)"
 
 usage="
-$PROGNAME [-b branch] [-r rundir] [-c config file] [-p parallelism factor]
+$PROGNAME [-b branch] [-j jar path] [-r rundir] [-c config file] [-p parallelism factor]
 
 Builds and runs specified branch of Cromwell and runs Centaur against it.
 
 Arguments:
-    -b    Branch of Cromwell to test
+    -b    Branch of Cromwell to test. Mutually exclusive with -j
+    -j    Path of a cromwell jar to use. Mutually exclusive with -b
     -r    Directory where script is run (defaults to current directory)
     -c    If supplied, the config file to pass to Cromwell
     -p    Number of simultaneous tests to run. Defaults to 3
 "
 
 INITIAL_DIR=$(pwd)
-
-PARALLELISM_FACTOR=3
-CROMWELL_BRANCH="develop"
 RUN_DIR=$(pwd)
-CONFIG_STRING=""
 
-while getopts ":hb:r:c:p:" option; do
+while getopts ":hb:r:c:p:j:" option; do
     case "$option" in
 	h) echo "$usage"
 	   exit
 	   ;;
 	b) CROMWELL_BRANCH=$OPTARG
 	   ;;
-        r) RUN_DIR=$OPTARG
-	   mkdir -p "${RUN_DIR}"
+    r) RUN_DIR=$OPTARG
+       mkdir -p "${RUN_DIR}"
+       ;;
+    c) CONFIG_STRING="-Dconfig.file=$OPTARG"
 	   ;;
-        c) CONFIG_STRING="-Dconfig.file=$OPTARG"
+    p) PARALLELISM_FACTOR=$OPTARG
 	   ;;
-        p) PARALLELISM_FACTOR=$OPTARG
+	j) CROMWELL_JAR=$OPTARG
```

Comment:
I thought this was an artifact of the diff as I don't see it in my intellij but it seems to be in the raw view here as well